src/applications/vaos/new-appointment/components/CommunityCareProviderSelectionPage/ProviderSelectionField.jsx

```diff
@@ -177,234 +184,235 @@ function ProviderSelectionField({
             </div>
           </section>
         )}
-      {showProvidersList && (
-        <>
-          <h2
-            id="providerSelectionHeader"
-            className="vads-u-font-size--h3 vads-u-margin-top--0"
-          >
-            Choose a provider
-          </h2>
-          {!loadingProviders &&
-            requestLocationStatus === FETCH_STATUS.succeeded &&
-            sortByDistanceFromCurrentLocation && (
-              <p className="vads-u-margin-top--0">
-                Providers based on your location
+        {showRemoveProviderModal && (
+          <RemoveProviderModal
+            provider={formData}
+            address={address}
+            onClose={response => {
+              setShowRemoveProviderModal(false);
+              if (response === true) {
+                setCheckedProvider(false);
+                onChange({});
+              }
+            }}
+          />
+        )}
+      </div>
+    );
+  }
+
+  return (
```

Comment:
The changes in here are hard to describe, since the diff isn't helpful. Primarily, I was trying to make the conditional checks less complicated.